tests/integration/ambient/main_test.go

```diff
@@ -88,7 +89,13 @@ func TestMain(m *testing.M) {
 			// can't deploy VMs without eastwest gateway
 			ctx.Settings().SkipVMs()
 			cfg.DeployEastWestGW = false
-		})).
+			cfg.ControlPlaneValues = `
+values:
+  ztunnel:
+    env:
+      SECRET_TTL: 300
+`
+		}, cert.CreateCASecret)).
```

Comment:
@hzxuzhonghu any concerns about the additional setup settings here affecting the entire ambient suite?  I know the [root cert rotation test](https://github.com/istio/istio/pull/47641) I was using as a basis runs in its own suite and uses the `Label(label.CustomSetup).` flag.  Ambient is already it's own suite, and I don't think there is a precedent of adding another ambient only test suite